app/code/Magento/ConfigurableProduct/view/adminhtml/web/js/variations/steps/select_attributes.js

```diff
@@ -130,6 +185,13 @@ define([
          * Back.
          */
         back: function () {
+        },
+
+        addProductManualy: function (data, event) {
+            productList().isShowAddProductButton(true);
+            this.variationsComponent().render(null, this.attributes());
+            this.modalComponent().closeModal();
+            $("[data-index='add_products_manually_button']").trigger('click');
```

Comment:
Triggering one click by another click does not seem to be a good solution. Can we avoid it somehow? I think we need to find a better solution.

Reply:
@dmytro-ch 
I try to do another way but i did not find solution as i see into product-grid lot of time trigger event call